schemas/people.yml

```diff
@@ -0,0 +1,14 @@
+title: str()
+icon: str()
+data: list(include('person'))
+---
+person:
+  name: str()
+  type: str()
+  team: str()
+  subteam: str()
+  title: str()
+  github_username: str()
+  brown_directory_uuid: str()
+  bio: str()
```

Comment:
bio isn't used - can be dropped.  And I think we should add image_hover (it's been implicit to this point, but I think it would be better to make explicit).